registration_system.cpp

```diff
@@ -0,0 +1,23 @@
+/*
+    Author: Alexis Osnaya.
+    URL: https://codeforces.com/problemset/problem/4/C
+*/
+
+#include <bits/stdc++.h>
+
+using namespace std;
+
+int main(){
+    int cases;
+    string str;
+    map<string,int> mp;
```

Comment:
Intenta ponerle un nombre que explique que va ser tu map o que va guardar, porque mp no se entiende que va hacer.

Reply:
Get it!!